tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -32,6 +32,7 @@ import {
   siNumberFormatter,
 } from '../../../widgets/line_chart_v2/lib/formatter';
 import {LineChartComponent} from '../../../widgets/line_chart_v2/line_chart_component';
+import {findClosestIndex} from '../../../widgets/line_chart_v2/sub_view/line_chart_interactive_utils';
```

Comment:
I might be hesitant to import the interactive_utils widget here but let's keep this right now and refactor later.